sdk/search/search-documents/src/generated/data/models/index.ts

```diff
@@ -108,7 +108,7 @@ export interface SearchRequest {
   orderBy?: string;
   /**
    * A value that specifies the syntax of the search query. The default is 'simple'. Use 'full' if
-   * your query uses the Lucene query syntax. Possible values include: 'simple', 'full'
+   * your query uses the Lucene query syntax. Possible values include: 'Simple', 'Full'
```

Comment:
is it expected that generated types now have capital first letters?

Reply:
Latest changes from autorest it seems. Looks incorrect. I will revert this and open an issue.